geotrek/api/v2/filters.py

```diff
@@ -960,7 +960,7 @@ class RelatedObjectsPublishedNotDeletedFilter(BaseFilterBackend):
 
     def filter_queryset_related_objects_published_not_deleted(self, qs, request, related_name, optional_query=Q()):
         # Exclude if no related objects exist
-        qs = qs.exclude(**{'{}'.format(related_name): None})
+        qs = qs.filter(**{'{}__isnull'.format(related_name): False})
```

Comment:
Bypasses this problem : https://code.djangoproject.com/ticket/26261

Reply:
add a comment for this. it seems to be fixed in next 4.2